### DISCUSSION TITLE
How to convert string to bytes?

### DISCUSSION THREAD

**Discussion by svend** [cuelang/cue#694] [Q&A] [answered]
How to convert `string` to `bytes`?

```
s: "a string"
b: bytes
b: s
```

```
_|_ // b: conflicting values bytes and "a string" (mismatched types bytes and string)
```

**Comment by svend** [comment, marked as answer]
Answer from @mpvl on Slack:

Use interpolation:

```
s: "a string"
b: bytes
b: '\(s)'
```

```
s: "a string"
b: 'a string'
```

**Comment by cueckoo** [comment]
This discussion has been migrated to https://github.com/cue-lang/cue/issues/694.

For more details about CUE's migration to a new home, please see https://github.com/cue-lang/cue/issues/1078.
